src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -76,12 +76,18 @@ void OnSystemRequestNotification::Run() {
   }
 
   if (RequestType::PROPRIETARY == request_type) {
-  std::string filename =
+    /* According to requirements:
```

Comment:
If the implemented behaviour is according to the requirements just remove it this disabled code below